testsuites/syncgateway/functional/topology_specific_tests/multiple_servers/test_muliple_servers.py

```diff
@@ -0,0 +1,333 @@
+from keywords import userinfo
+
+
+@pytest.mark.sanity
+@pytest.mark.syncgateway
+@pytest.mark.changes
+@pytest.mark.rebalance
+@pytest.mark.skip(reason="Failing due to - https://github.com/couchbase/sync_gateway/issues/2173")
+def test_rebalance_sanity(params_from_base_test_setup):
+
+    cluster_config = params_from_base_test_setup["cluster_config"]
+    mode = params_from_base_test_setup["mode"]
+
+    cluster_helper = ClusterKeywords()
+
+    sg_conf_name = "sync_gateway_default_functional_tests"
+    sg_conf_path = sync_gateway_config_path_for_mode(sg_conf_name, mode)
+
+    cluster_helper.reset_cluster(cluster_config=cluster_config,
+                                 sync_gateway_config=sg_conf_path)
+
+    topology = cluster_helper.get_cluster_topology(cluster_config)
+
+    admin_sg_one = topology["sync_gateways"][0]["admin"]
+    sg_one_url = topology["sync_gateways"][0]["public"]
+
+    cluster_servers = topology["couchbase_servers"]
+    cbs_one_url = cluster_servers[0]
+    cbs_two_url = cluster_servers[1]
+
+    log_info("Running: 'test_distributed_index_rebalance_sanity'")
+    log_info("cluster_config: {}".format(cluster_config))
+    log_info("admin_sg: {}".format(admin_sg_one))
+    log_info("sg_url: {}".format(sg_one_url))
+    log_info("cbs_one_url: {}".format(cbs_one_url))
+    log_info("cbs_two_url: {}".format(cbs_two_url))
+
+    sg_db = "db"
+    num_docs = 100
+    num_updates = 100
+    sg_user_name = "seth"
+    sg_user_password = "password"
+    channels = ["ABC", "CBS"]
+
+    client = MobileRestClient()
+    cb_server = CouchbaseServer(cbs_one_url)
+    server_to_remove = CouchbaseServer(cbs_two_url)
+
+    client.create_user(admin_sg_one, sg_db, sg_user_name, sg_user_password, channels=channels)
+    session = client.create_session(admin_sg_one, sg_db, sg_user_name)
+
+    with concurrent.futures.ThreadPoolExecutor(5) as executor:
+
+        # Add docs to sg
+        log_info("Adding docs to sync_gateway")
+        docs = client.add_docs(sg_one_url, sg_db, num_docs, "test_doc", channels=channels, auth=session)
+        assert len(docs) == num_docs
+
+        # Start updating docs and rebalance out one CBS node
+        log_info("Updating docs on sync_gateway")
+        update_docs_task = executor.submit(client.update_docs, sg_one_url, sg_db, docs, num_updates, auth=session)
+
+        # Run rebalance in background
+        cb_server.rebalance_out(cluster_servers, server_to_remove)
+
+        updated_docs = update_docs_task.result()
+        log_info(updated_docs)
+
+    # Verify docs / revisions present
+    client.verify_docs_present(sg_one_url, sg_db, updated_docs, auth=session)
+
+    # Verify docs revisions in changes feed
+    client.verify_docs_in_changes(sg_one_url, sg_db, updated_docs, auth=session)
+
+    # Rebalance Server back in to the pool
+    cb_server.add_node(server_to_remove)
+    cb_server.rebalance_in(cluster_servers, server_to_remove)
```

Comment:
Is this rebalance just to restore the cluster state prior to the subsequent test?  (and that's why we don't do any additional verification post-restore?)